python/torch_mlir_e2e_test/test_suite/stats.py

```diff
@@ -66,8 +71,31 @@ def forward(self, x):
 def MeanDtypeModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4, 5).to(torch.float64))
 
+
+# ==============================================================================
+
+
+class MeanLargeInputModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1, -1], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.mean(x)
+
+
+@register_test_case(module_factory=lambda: MeanLargeInputModule())
+def MeanLargeInputModule_basic(module, tu: TestUtils):
+    module.forward(100 + tu.rand(3, 4, 1024, 8192))
```

Comment:
nit: use `low, high` to specify custom range rather than adding. This applies to the other tests below as well